src/components/ProjectContributors.js

```diff
@@ -129,122 +130,15 @@ const ProjectContributors = (props) => {
-                        {
-                            activeContributorsAllocated.length != 0
-                                ? renderContributors({
-                                    active: true,
-                                    contributors: activeContributorsAllocated,
-                                    project: project
-                                })
-                                : <ContributorsEmptyState active/>
-
-                        }
-                    </Grid>
-                </Box>
-                <Box my={[2, 5]}>
-                    <Typography align='left' variant='h5'>
-                        {`Upcoming contributors`}
-                    </Typography>
-                    <Grid container>
-                        {
-                            upcomingContributorsAllocatedOnly.length != 0
-                                ? renderContributors({
-                                    active: true,
-                                    contributors: upcomingContributorsAllocatedOnly,
-                                    project: project
-                                })
-                                : <ContributorsEmptyState active/>
-
-                        }
-                    </Grid>
-                </Box>
-                <hr/>
-            </Grid>
-            <h1>
-                {`Add new contributors to the project`}
-            </h1>
-            {errorGithubContributors
-                ? isNotAContributor()
-                : null
-            }
-            <Grid item xs={12}>
-                <Box>
-                    <Grid container>
-                        {
-                            contributorsToAdd.length != 0
-                                ? renderContributors({
-                                    active: false,
-                                    contributors: contributorsToAdd,
-                                    project: project
-                                })
-                                : <ContributorsEmptyState/>
-                        }
-                    </Grid>
-                </Box>
-                <Box my={[2, 5]} py={5}/>
-            </Grid>
-            <Grid item xs={12}>
-                {
-                    contributorClicked &&
-                    <AllocationAddForm
-                        project={project}
-                        open={openAddAllocationDialog}
-                        onClose={handleAddAllocationClose}
-                        contributor={contributorClicked}
-                    />
-                }
-            </Grid>
-        </Grid>
+        <ContributorList 
+            contributors={contributors}
+            project={project}
+            activeContributorsAllocated={activeContributorsAllocated}
+            upcomingContributorsAllocatedOnly={upcomingContributorsAllocatedOnly}
+            errorGithubContributors={errorGithubContributors}
+            isNotAContributor={isNotAContributor}
```

Comment:
@Ovifer13 Instead of passing this from the parent, we can just import the component `GithubAccessBlocked` directly in the `ContributorList` component and with a conditional render the component itself, I don't remember why there was a function to return the component but I think that could be changed